packages/ember-runtime/lib/mixins/array.js

```diff
@@ -936,17 +932,16 @@ const ArrayMixin = Mixin.create(Enumerable, {
     @method reduce
     @param {Function} callback The callback to execute
     @param {Object} initialValue Initial value for the reduce
-    @param {String} reducerProperty internal use only.
```

Comment:
`reducerProperty` seems not used anywhere and remains from SproutCore days I think